docfiles/head.html

```diff
@@ -1,8 +1,8 @@
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
 <meta name="viewport" content="width=device-width,height=device-height,user-scalable=no,initial-scale=1.0,maximum-scale=1.0,minimum-scale=1.0">
 
-<link rel="stylesheet" href="/doccdn/semantic.css" />
-<link rel="stylesheet" href="/doccdn/blockly.css" />
+<link rel="stylesheet" href="/blb/semantic.css" />
+<link rel="stylesheet" href="/blb/blockly.css" />
```

Comment:
Also if it's not in that replFiles list, how is it replacing doccdn in the first place for head.html?